app/assets/javascripts/lib/pulltorefresh.js

```diff
@@ -22,7 +22,7 @@
     distThreshold: 60,
     distMax: 80,
     distReload: 50,
-    distIgnore: 0,
+    distIgnore: 10,
```

Comment:
This slightly improves the general behavior of PTR making it a little bit "delayed".